assets/style/app/components/_page-footer.scss

```diff
@@ -73,20 +76,34 @@
 
     .links-social li {
       display: inline-block;
-      margin: 0 $space-base/2;
-      vertical-align: middle;
+      vertical-align: webkit-baseline-middle;
+      margin-right: 15px;
```

Comment:
can you make this margin based off of the `$space` widths that are available please?